src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
+        System.out.println(ball1.getRandomBall().toString());
+        Lottery ball2 = new Lottery();
+        System.out.println(ball2.getRandomBall().toString());
+        Lottery ball3 = new Lottery();
+        System.out.println(ball3.getRandomBall().toString());
     }
```

Comment:
In the description task said,  need the create three balls. The object class Lottery returns object Ball in the method getRandomBall().
This code don't need:
```
        Lottery ball1 = new Lottery();
        System.out.println(ball1.getRandomBall().toString());
        Lottery ball2 = new Lottery();
        System.out.println(ball2.getRandomBall().toString());
        Lottery ball3 = new Lottery();
        System.out.println(ball3.getRandomBall().toString());
```
we need create one object the class Lottery:
`Lottery lottery = new Lottery();`
and use the method getRandomBall() for create three objects Ball:
```
int index = 1;
while(index <= 3){
    System.out.println(lottery.getRandomBall().toString());
    index +=1;
}

```